examples/font.rs

```diff
@@ -0,0 +1,31 @@
+//! An example of drawing text. Writes to the user-provided target file.
+
+extern crate image;
+extern crate imageproc;
+extern crate rusttype;
+
+use std::path::Path;
+use imageproc::pixelops::weighted_sum;
+use image::{Rgb, RgbImage, open, ImageBuffer, GenericImage};
+use rusttype::{FontCollection, Scale, point, PositionedGlyph};
+
```

Comment:
You use most of them in the implementation of draw_text, but not in this file (weighted_sum, for example, isn't used).